modules/apps/marketplace/marketplace-store-web/src/main/java/com/liferay/marketplace/store/web/internal/portlet/RemoteMVCPortlet.java

```diff
@@ -75,46 +73,6 @@
  */
 public class RemoteMVCPortlet extends MVCPortlet {
 
-	public void authorize(
```

Comment:
@slnn @dantewang I do not think you can remove this, cause there is usage in jsp like
`<liferay-portlet:actionURL name="authorize" var="authorizeURL">
		<portlet:param name="callbackURL" value="<%= callbackURL %>" />
	</liferay-portlet:actionURL>`

@vicnate5 This pull passed on ci, I am wondering if we have test related to marketplace authorization?

Reply:
Going to run tests at https://github.com/vicnate5/liferay-portal/pull/2174. Ideally I can find this breaking something so I know we have coverage.

Reply:
Confirmed with automation tests that this breaks Marketplace. No errors logged, but the Marketplace app does not render.